src/strutil.c

```diff
@@ -371,7 +371,7 @@ SR_PRIV int sr_atof_ascii(const char *str, float *ret)
 		return SR_ERR;
 	}
 
-	/* FIXME This fails unexpectedly. Some other method to safel downcast
+	/* FIXME This fails unexpectedly. Some other method to safely downcast
 	 * needs to be found. Checking against FLT_MAX doesn't work as well. */
 	/*
 	if ((float) tmp != tmp) {
```

Comment:
Can you split the `src/serial.c` and `src/strutil.c` patches up into a separate "small fixes" patch?